docs/source/build_a_lint_rule.ipynb

```diff
@@ -130,6 +130,50 @@
     "Iteration order of attributes is the same as the order they appear in the source code.\n",
     "In this case, that means visit_If_test is called before visit_If_body and visit_If_orelse.\n",
     "\n",
+    "Use fixit's cli to generate a skeleton of adding a new rule file::\n",
+    "\n",
+    "    $ python -m fixit.cli.add_new_rule # Creates new_rule.py at fixit/rules/new_rule.py\n",
+    "    $ python -m fixit.cli.add_new_rule --path fixit/rules/my_rule.py # Creates rule file at path specified\n",
+    "\n",
+    "This will generate a model rule file used to create new rule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import libcst as cst\n",
+    "import libcst.matchers as m\n",
+    "\n",
+    "from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid\n",
+    "\n",
+    "class Rule(CstLintRule):\n",
+    "    \"\"\"\n",
+    "    docstring or new_rule description\n",
+    "    \"\"\"\n",
+    "\n",
+    "    MESSAGE = \"Enter rule description message\"\n",
+    "\n",
+    "    VALID = [Valid(\"'example'\")]     # Valid examples\n",
+    "\n",
+    "    INVALID = [Invalid(\"'example'\")] # Invalid examples\n",
+    "    ..."
+   ]
+  },
```

Comment:
The changes you added to Getting Started should just replace this part. So whenever lint rule template changes, we don't need to manually update this example.
Getting Started is not need to be updated.

Before commit change, please click on "Restart & Clear Output". So all output are not stored.
The CI job will run through each cell and generate the result on the fly.
![image](https://user-images.githubusercontent.com/3840867/92777925-878a9f00-f355-11ea-94c3-f7c86529eceb.png)

Reply:
Got it. I thought, we have to run those cells in order to show output.